Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -4650,6 +4650,22 @@ void RenderLayerCompositor::rootOrBodyStyleChanged(RenderElement& renderer, cons
     }
 }
 
+void RenderLayerCompositor::setRootElementCapturedInViewTransition(bool captured)
+{
+    m_rootElementCapturedInViewTransition = captured;
```

Comment:
Nit - do we need to call `updateRootContentsLayerBackgroundColor()` if the value doesn't change?

```suggestion
    if (m_rootElementCapturedInViewTransition == captured)
        return;

    m_rootElementCapturedInViewTransition = captured;
```